open-apparel-registry/open-apparel-registry
# Update embedded map logo

## Overview

This PR updates the embedded map logo.

It also fixes the `useFooterHeight` hook which used the wrong class name.

Connects #2186 

## Demo

<img width="1410" alt="Screen Shot 2022-10-13 at 4 04 20 PM" src="https://user-images.githubusercontent.com/8356789/195710262-0c5af208-cec8-4ef4-ae52-bc6d11808cee.png">

<img width="777" alt="Screen Shot 2022-10-13 at 4 05 11 PM" src="https://user-images.githubusercontent.com/8356789/195710431-f914e16c-accd-4e22-8706-fb951693ed58.png">

## Testing Instructions

* Load the embedded maps page
* Ensure the "Powered By" logo is updated and is styled correctly in both desktop and mobile

## Checklist
